Documentation/ceph-cluster-crd.md

```diff
@@ -39,12 +39,16 @@ For more details on the mons and when to choose a number other than `3`, see the
   - [storage selection settings](#storage-selection-settings)
   - [storage configuration settings](#storage-configuration-settings)
 
-#### Node updates
+#### Node Updates
 Nodes can be added and removed over time by updating the Cluster CRD, for example with `kubectl -n rook-ceph edit cluster rook`.
 This will bring up your default text editor and allow you to add and remove storage nodes from the cluster.
 This feature is only available when `useAllNodes` has been set to `false`.
 
-### Node settings
+### Mon Settings
+- `count`: set the number of mons to be started. The number should be odd and between `1` and `9`. Default if not specified is `3`.
+- `allowMultiplePerNode`: enable or disable the placement of multiple mons on one node. Default is `true`.
```

Comment:
I'd suggest default to `false` for production settings

Reply:
The reason why I chose `true` as the default, is because this could potentially break (small) existing cluster that would be upgraded.
Should I leave it at `true` or `false`?

Reply:
i'd suggest the defaults and documentation be for production, but the examples work for test environments. So a default of `false` sounds good, then you don't need to worry about a pointer either.